src/store/accountActions.js

```diff
@@ -132,12 +133,14 @@ export function importWallet(wallet, name, description) {
                         type: 'ACCOUNT/IMPORT_WALLET',
                         accountId: result,
                     });
-                    dispatch(loadAccountBalance(result));
-                    if (address(result))
+                    // We should load balance and add the new account to the canonical list.
+                    if (address(result)) {
+                        dispatch(addAddress(result, name, description));
```

Comment:
actually there should be an  `ADD_ACCOUNT` dispatch like in `createAccount` but i see we forgot to add it. the reducer will then add it to `accountsList`

```Javascript
dispatch({
                type: 'ACCOUNT/ADD_ACCOUNT',
                accountId: result,
                name,
                description,
            });
```